p2p/src/peer_manager/peers_eviction/mod.rs

```diff
@@ -194,16 +186,60 @@ pub fn select_for_eviction_inbound(
     find_group_most_connections(candidates)
 }
 
-#[must_use]
+/// Outbound block relay connections younger than this age will not be taken into account
+/// during eviction.
+/// Note that extra block relay connections are established and evicted on a regular basis
+/// during normal operation. So, this interval basically determines how often those extra
+/// connections will come and go.
+const BLOCK_RELAY_CONNECTION_MIN_AGE: Duration = Duration::from_secs(120);
```

Comment:
I thought we're trying to get out of the excessive use of constants, and put them in configuration structs. I was happy we're moving towards that.

Reply:
I renamed `ConnectionCountLimits` to `PeerManagerConfig` and put the new fields there.